### DISCUSSION TITLE
Getting metrics from OpenSTA?

### DISCUSSION THREAD

**Discussion by donn** [The-OpenROAD-Project/OpenROAD#3199] [Q&A] [answered]
The `-metrics` feature is useful, but I wonder if there is a way to get it when using the `sta` binary that's included with OpenROAD. We use this binary as OpenROAD's sta commands do not support hierarchical STA, so I'm just wondering if this is at all possible.

**Comment by maliberty** [comment, marked as answer]
All the metrics generation is in OR so I don't think so.
